MatchingGame.js

```diff
@@ -2,12 +2,10 @@ const section = document.querySelector("section");
 const playerLivesCount = document.querySelector("span");
 let playerLives = 10;
 
-
 // Link text
 playerLivesCount.textContent = playerLives;
 
-
-//Generate the data
```

Comment:
Always put a space after the `//` when writing comments. Also always put comments above the line you are trying to add info to (you didn't do this but wanted to lyk as well)